lib/libtrace.h.in

```diff
@@ -724,7 +724,7 @@ typedef struct libtrace_packet_t {
 	int error;                      /**< The error status of pread_packet */
         uint64_t internalid;            /** Internal identifier for the pkt */
         void *srcbucket;                /** Source bucket in trace_rt or stream in XDP */
-
+        void *fmtdata;                  /**< Storage for format-specific data */
```

Comment:
Is ```srcbucket``` in _trace\_rt_ essentially being used in the same way as ```fmtdata```?
Do we need both ```fmtdata``` and ```srcbucket```?

If both aren't needed, this could be a union, or ```srcbucket``` could be renamed to ```fmtdata```.
If both are needed, then I'll update XDP to use ```fmtdata``` because it is currently hijacking ```srcbucket```.